scripts/examples-install.sh

```diff
@@ -7,3 +7,4 @@ npm install --no-optional
 (cd public/docs/_examples/_protractor && npm install --no-optional)
 npm run webdriver:update --prefix public/docs/_examples/_protractor
 gulp add-example-boilerplate
+sh -e /etc/init.d/xvfb start
```

Comment:
Problem with this is that I couldn't use this script anymore on my machine to install the repo, right? Couldn't we leave this on the travis file?